frontend/src/pages/TransactionLink.spec.js

```diff
@@ -101,7 +101,7 @@ describe('TransactionLink', () => {
               validUntil: transactionLinkValidExpireDate(),
               redeemedAt: '2022-03-18T10:08:43.000Z',
               deletedAt: now,
-              user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+              user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```

Comment:
isn't there a validation on the gradidoID if it has the correct format and version?